main.go

```diff
@@ -75,14 +75,20 @@ func generate(name string, args []string) {
 		log.Fatalln("error parsing version:", err)
 	}
 
-	addExtRegexp, err := regexp.Compile(*addext)
-	if err != nil {
-		log.Fatalln("error parsing extension inclusion regexp:", err)
+	var addExtRegexp *regexp.Regexp = nil
+	if *addext != "" {
```

Comment:
Arg, serves me right for doing this in a hurry. Created #84 to update the flag docs. This time I've learned my lesson, I'll leave it to you to review and merge.